### ISSUE TITLE
Bug using slash in string 

### ISSUE THREAD

**Issue by LoiKos** [Thom1729/Sublime-JS-Custom#56]
i have a bug using a / in a string that break the syntax coloration :

```javascript 
 let type = null
    if (bill) {
      type = 'Ex/ample' // here syntax isn't render correctly 
    } else if (vn) {
      type = 'Something'
    } else if (ctn) {
      type = 'Something else'
    } else if (imo) {
      type = 'Something else'
    }
```

![capture du 2018-12-24 11-24-29](https://user-images.githubusercontent.com/9548473/50397162-cbe39f00-076e-11e9-8389-d4d6bc380b1d.png)

**Comment by mohitsinghs** [comment]
Okay, I was able to reproduce this.
It appears like this is not a bug, and instead a feature.
I'm not using [flow](https://flow.org) but as I looked into [docs](https://flow.org/en/docs/types/aliases), the culprit is your `type` variable, which is being treated as flow type annotation.
Now there are two thing
 - Look for [`flow_types` option ](https://github.com/Thom1729/Sublime-JS-Custom#flow_types-boolean) in your JSCustom User Settings and remove that (Only if you are not using flow)
 - Or, avoid using `type` as a variable name.

I hope this helps.

**Comment by Thom1729** [comment]
When the flow syntax extension sees `type` at the beginning of the line, it assumes that it's a type annotation. This should be an easy fix. If I don't get to it today due to the holidays, it should be fixed by Wednesday.

**Comment by Thom1729** [comment]
By the way, in the immediate term you can work around this by wrapping the statement in parens: 

```js
(type = 'Ex/ample');
```

**Comment by Thom1729** [comment]
v1.0.28 should fix this. Please let me know if it does not.

**Comment by LoiKos** [comment]
it's fix with v1.0.28 :+1:
